src/service/viewer-impl.js

```diff
@@ -154,9 +154,6 @@ export class Viewer {
     /** @private {?function(string, *, boolean):(Promise<*>|undefined)} */
     this.messageDeliverer_ = null;
 
-    /** @private {?string} */
-    this.messagingOrigin_ = null;
```

Comment:
We need to keep `this.messaingOrigin_` var intact - it will be needed for other purposes. We'll simply create a new set of tools for viewer origin.